__tests__/app.test.js

```diff
@@ -93,21 +122,22 @@ describe("app ", () => {
           expect(review.body[0].review_img_url).toBe(testData.reviewData[0].review_img_url);
         });
     });
-    test("a review object must have, review_id,title,review_body,designer,review_img_url,votes,category,owner,created_at - property", () => {
+    test("a review object must have, review_id,title,review_body,designer,review_img_url,votes,category,owner,created_at,comment_count - property", () => {
       return request(app)
         .get("/api/reviews/1")
         .expect(200)
         .then((review) => {
           review.body.forEach((review) => {
-            expect(review).toHaveProperty("review_id");
-            expect(review).toHaveProperty("title");
-            expect(review).toHaveProperty("category");
-            expect(review).toHaveProperty("designer");
-            expect(review).toHaveProperty("owner");
-            expect(review).toHaveProperty("review_body");
-            expect(review).toHaveProperty("review_img_url");
-            expect(review).toHaveProperty("created_at");
-            expect(review).toHaveProperty("votes");
+            expect(review).toHaveProperty("review_id", expect.any(Number));
```

Comment:
as this is a specific review, the values of which you already know, it's preferable to just test that the object equals all the values you expect for review #1